src/algorithms/mazeGenerators/Maze.java

```diff
@@ -13,19 +20,21 @@
  * mazeArr - int array for the maze's presentation
  * posArr - Array of positions holding each position at it's place.
  */
-public class Maze {
+public class Maze implements Serializable{
     private int rows;
     private int columns;
     private Position StartPosition;
     private Position GoalPosition;
     private int[][] mazeArr;
     private Position[][] posArr;
+    private byte[] bytes;
```

Comment:
where do we use this?

Reply:
In the readObject method that overrides the objectInputStream. You can see the function below.